src/storage/test/GetNeighborsBenchmark.cpp

```diff
@@ -0,0 +1,317 @@
+        nebula::cpp2::ColumnDef column1;
+        column1.type.type = nebula::cpp2::SupportedType::STRING;
+        schema->addField("playerName", std::move(column1.type));
+
+        nebula::cpp2::ColumnDef column2;
+        column2.type.type = nebula::cpp2::SupportedType::STRING;
+        schema->addField("teamName", std::move(column2.type));
+
+        nebula::cpp2::ColumnDef column3;
+        column3.type.type = nebula::cpp2::SupportedType::INT;
+        schema->addField("startYear", std::move(column3.type));
+
+        nebula::cpp2::ColumnDef column4;
+        column4.type.type = nebula::cpp2::SupportedType::INT;
+        schema->addField("endYear", std::move(column4.type));
+
+        nebula::cpp2::ColumnDef column5;
+        column5.type.type = nebula::cpp2::SupportedType::INT;
+        schema->addField("teamCareer", std::move(column5.type));
+
+        nebula::cpp2::ColumnDef column6;
+        column6.type.type = nebula::cpp2::SupportedType::INT;
+        schema->addField("teamGames", std::move(column6.type));
+
+        nebula::cpp2::ColumnDef column7;
+        column7.type.type = nebula::cpp2::SupportedType::DOUBLE;
+        schema->addField("teamAvgScore", std::move(column7.type));
+
+        nebula::cpp2::ColumnDef column8;
+        column8.type.type = nebula::cpp2::SupportedType::STRING;
+        schema->addField("type", std::move(column8.type));
+
+        nebula::cpp2::ColumnDef column9;
+        column9.type.type = nebula::cpp2::SupportedType::INT;
+        schema->addField("champions", std::move(column9.type));
+
+        schemaMan->addEdgeSchema(spaceId, 101, schema);
+    }
+
+    mockData(gKV.get());
+}
+
+cpp2::GetNeighborsRequest buildRequest(const std::vector<VertexID> vIds,
+                                       const std::vector<std::string>& playerProps,
+                                       const std::vector<std::string>& serveProps) {
+    GraphSpaceID spaceId = 0;
+    PartitionID partId = 0;
+    TagID tagId = 1;
+    EdgeType edgeType = 101;
+
+    cpp2::GetNeighborsRequest req;
+    req.set_space_id(spaceId);
+    decltype(req.parts) parts;
+    parts[partId] = vIds;
+    req.set_parts(std::move(parts));
+    req.set_edge_types({edgeType});
+
+    for (const auto& prop : playerProps) {
+        req.return_columns.emplace_back(TestUtils::vertexPropDef(prop, tagId));
+    }
+    for (const auto& prop : serveProps) {
+        req.return_columns.emplace_back(TestUtils::edgePropDef(prop, edgeType));
+    }
+    return req;
+}
+
+}  // namespace storage
+}  // namespace nebula
+
+void go(int32_t iters,
+        const std::vector<nebula::VertexID> vIds,
+        const std::vector<std::string>& playerProps,
+        const std::vector<std::string>& serveProps) {
+    nebula::storage::cpp2::GetNeighborsRequest req;
+    BENCHMARK_SUSPEND {
+        req = nebula::storage::buildRequest(vIds, playerProps, serveProps);
+    }
+    auto executor = std::make_unique<folly::CPUThreadPoolExecutor>(1);
```

Comment:
I think we should build the thread pool outside the test case.

Reply:
my bad, fixed.